cortx-motr.spec.in

```diff
@@ -174,7 +193,13 @@ CORTX Motr unit tests (for kernel and user space).
 %setup -q
 
 %build
-%configure %{with_linux} %{with_lustre} %{configure_opts}
+%if  "%{CONFIG_ARCH_TYPE}"=="x86_64"
+#x86_64 platform
+%configure %{with_linux} %{with_lustre} %{configure_opts} 
+%else		
+#aarch64 platform
+%configure %{with_linux} %{with_lustre} %{configure_opts} enable_castxml=yes  
+%endif
```

Comment:
As castxml is manually set in motr build pipeline, the flag "enable_castxml=yes" is required to make the  build aware off that it is going to built against castxml rather than gccxml as gccxml is the default choice for the motr build